README.md

```diff
@@ -2,239 +2,293 @@
+### Option #1 (Recommended) -  Project-centric approach
 
-# example build command
-> cmake --build c:/path/to/o3de-multiplayersample/build/windows_vs2019 --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher --config profile -- /m /nologo
-```
+This option outputs all the project binaries in the project's build directory (for example `c:/my-o3de-projects/o3de-multiplayersample/build`).
 
-#### Option #2 - Engine-centric approach to building a project
+1. Example project-centric configure command.
 
-This option will output all the project and engine binaries in the engine's build folder e.g. c:/path/to/o3de/build
+   ```shell
+   cmake -S c:/my-o3de-projects/o3de-multiplayersample -B c:/my-o3de-projects/o3de-multiplayersample/build/windows -G "Visual Studio 16" -DLY_3RDPARTY_PATH="c:/o3de-packages"
+   ```
 
-```shell
-# example configure command
-> cmake -S c:/path/to/o3de -B c:/path/to/o3de/build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH="c:/3rdparty" -DLY_PROJECTS="c:/path/to/o3de-multiplayersample"
+1. Example project-centric build command.
 
-# example build command
-> cmake --build c:/path/to/o3de/build/windows_vs2019 --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher --config profile -- /m /nologo
-```
+   ```shell
+   cmake --build c:/my-o3de-projects/o3de-multiplayersample/build/windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher --config profile -- /m /nologo
+   ```
 
-### Step 4. Setup Client and Server
+### Option #2 - Engine-centric approach to building a project
 
-Under project root, there should be two files: `launch_client.cfg` and `launch_server.cfg`. 
+This option will output all the project and engine binaries in the engine's build directory (for example, `c:/o3de/build`).
 
-File `launch_client.cfg` should contain:
-```shell
-connect
-```
+1. Example engine-centric configure command.
+
+   ```shell
+   cmake -S C:/o3de -B C:/o3de/build/windows -G "Visual Studio 16" -DLY_3RDPARTY_PATH="C:/o3de-packages" -DLY_PROJECTS="C:/o3de/o3de-multiplayersample"
+   ```
+
+1. Example engine-centric build command.
+
+   ```shell
+   cmake --build C:/o3de/build/windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher --config profile -- /m /nologo
+   ```
+
+## Step 4. Setup the client and server
+
+Under project root, there are two files: `launch_client.cfg` and `launch_server.cfg`.
+
+1. `launch_client.cfg` contains the client connection setting. To connect to a server that is running locally, add the following line:
+
+   ```shell
+   connect
+   ```
+
+   To connect to a remote server, add the IP address of the server after the connect statement. For example:
+
+   ```shell
+   connect 192.168.0.20
+   ```
+
+2. `launch_server.cfg` contains the initial level to load:
+
+   ```shell
+   LoadLevel Levels/NewStarbase/NewStarbase.spawnable
+   ```
+
+## Step 5. Launch the server
```

Comment:
We should probably mention that you need to cd C:\path\to\o3de-multiplayersample\build\windows\bin\profile here. See https://github.com/o3de/o3de-multiplayersample/issues/190

Reply:
Added the path to the command.